metadata.txt

```diff
@@ -19,14 +19,25 @@ email=tim@kartoza.com
 # Optional items:
 
 # Uncomment the following line and add your changelog:
-# changelog=
+changelog=
+	v2.0.0:
+	- Code converted to Python 3
+	- Plugin can now be installed in QGIS 3
+	- Fixed a language translation issue
+	- Processing can correctly be applied to multilinestring files
```

Comment:
Processing can mean a different thing in QGIS see https://docs.qgis.org/3.4/en/docs/user_manual/processing/index.html, may be we can explain that there are updates in the stream extraction logic to accommodate features with multilinestrings geometry.